packages/fxa-settings/src/components/DataBlock/index.tsx

```diff
@@ -47,11 +51,12 @@ export const DataBlock = ({
           valueIsArray ? 'py-4' : 'py-5'
         }`}
         data-testid={dataTestId}
+        onCopy={(e) => { onCopy && onCopy(e) }}
```

Comment:
Since `onCopy` is already optional on HTMLElements we should be able to just do `{...onCopy}` here

Reply:
Thanks, I'll change this. I might have some bad habits of doing non-short hand things. Keep this feedback coming :smile: